### PR TITLE
Claimant copy changes

### PR DESCRIPTION
**Pull request by leikkisa**
connects #8746

### CODE REVIEW

File: client/app/intake/components/SelectClaimant.jsx [hunk 1]
```diff
@@ -32,9 +32,15 @@ export default class SelectClaimant extends React.PureComponent {
...
 
-    const claimantLabel = 'Please select the claimant listed on the form. ' +
-    'If you do not see the claimant in the options below, add them in VBMS, ' +
-    'then refresh this page.';
+    const claimantTextStart = 'Please select the claimant listed on the form. ' +
+    'If you do not see the claimant in the options below, and you have access, ' +
+    'please add the claimant to the Corporate Database to continue processing this intake. ' +
+    'If you do not have access, please ';
+    const email = React.createElement(
+      'a', { href: 'mailto:jennifer.umberhind@va.gov?Subject=Add%20claimant%20to%20Corporate%20Database' }, 'email'
```
Comment (pkarman) [inline client/app/intake/components/SelectClaimant.jsx]: hardcoding personal emails in code always feels yucky to me. can we get an address that isn't specific to a human?

could we make the string a constant?

also I notice that `client/COPY.json` exists and that might be a place to hold all this kind of thing?

File: client/app/intake/components/SelectClaimant.jsx [hunk 2]
```diff
@@ -78,8 +84,10 @@ export default class SelectClaimant extends React.PureComponent {
 
       { showClaimants && hasRelationships && claimantOptions() }
       { showClaimants && !hasRelationships && <p className="cf-red-text">
-        The Veteran has no relationships in our records. You can add the relationship
-        in VBMS and refresh this page.
+        This Veteran currently has no known relationships. If you have access, please add the claimant to the
+        Corporate Database to continue processing this intake. If you do not have access, please
+        <a href="mailto:jennifer.umberhind@va.gov?Subject=Add%20claimant%20to%20Corporate%20Database"> email </a>
```
Comment (pkarman) [inline client/app/intake/components/SelectClaimant.jsx]: same comment here

File: client/app/intake/components/SelectClaimant.jsx (hunk 1 above)
Comment (leikkisa) [reply client/app/intake/components/SelectClaimant.jsx]: Ken has proposed intake-help@va.gov on the last ticket like this too.  My understanding was that this was temporary until that was available. Let me check in with Steve/Carola/Ken to see if there has been movement on that.

I didn't know about the copy JSON! I will look into that.

Comment (leikkisa) [reply client/app/intake/components/SelectClaimant.jsx]: Linking related ticket to email thing here (will keep track of progress on that there): https://github.com/department-of-veterans-affairs/caseflow/issues/8316

**Comment by leikkisa** [comment]
I have started a conversation about making a non-personal email, will track progress on that here:
https://github.com/department-of-veterans-affairs/caseflow/issues/8316

For now, we could 1) move forward with this (Jennifer's email is also already in use on the search page) until the email is ready, 2) wait for the listserv, but that could take a little while, or 3) Lisa has kindly let us use the VACaseflowOps email that's already set up, and we could set up a subject-line based autoforward, or just use that temporarily until the other listserv is ready.

File: client/COPY.json [hunk 3]
```diff
@@ -321,5 +321,9 @@
   "STATUS_INQUIRY_MAIL_TASK_LABEL": "Status inquiry",
   "VACATE_MOTION_MAIL_TASK_LABEL": "Motion to vacate",
 
-  "CORRECT_REQUEST_ISSUES_LINK": "Correct issues"
+  "CORRECT_REQUEST_ISSUES_LINK": "Correct issues",
+
+  "NO_RELATIONSHIPS": "This Veteran currently has no known relationships. If you have access, please add the claimant to the Corporate Database to continue processing this intake. If you do not have access, please",
```
Comment (pkarman) [inline client/COPY.json]: 🎉

**Pull request merged**